superduperdb/ext/openai/model.py

```diff
@@ -34,6 +34,14 @@ def _available_models():
 
 @dc.dataclass(kw_only=True)
 class _OpenAI(APIModel):
+    '''
+    :param client_kwargs: The kwargs to be passed to OpenAI
+    :param async_client_kwargs: The kwargs to be passed to AsyncOpenAI
+
+    '''
+
+    client_kwargs: t.Optional[dict] = {}
```

Comment:
Should be a factory - shouldn't add mutable arguments as defaults.

Reply:
yes, saw in ci, I will fix

Reply:
done